oracle/datadog_checks/oracle/oracle.py

```diff
@@ -88,6 +95,25 @@ def validate_config(self):
         if not self._server or not self._user:
             raise ConfigurationError("Oracle host and user are needed")
 
+        if self._protocol.upper() not in VALID_PROTOCOLS:
+            raise ConfigurationError("Protocol %s is not valid, must either be TCP or TCPS" % self._protocol)
+
+        if (
+            self._jdbc_driver
+            and self._protocol.upper() == "TCPS"
+            and not (self._jdbc_truststore_type and self._jdbc_truststore_path)
+        ):
+            raise ConfigurationError(
+                "TCPS connections to oracle via JDBC requires both `jdbc_truststore_type` and `jdbc_truststore_path` "
+                "configuration options "
+            )
+
+        if self._jdbc_truststore_type and self._jdbc_truststore_type.upper() not in VALID_TRUSTSTORE_TYPES:
+            raise ConfigurationError(
+                "Truststore type %s is not valid, must be one of %s"
+                % (self._jdbc_truststore_type, VALID_TRUSTSTORE_TYPES)
+            )
```

Comment:
should this be nested to check for jdbc driver and tcps?

```
  if self._jdbc_driver and self._protocol.upper() == "TCPS":
       if not (self._jdbc_truststore_type and self._jdbc_truststore_path)
        ):
            raise ConfigurationError(
                "TCPS connections to oracle via JDBC requires both `jdbc_truststore_type` and `jdbc_truststore_path` "

       if self._jdbc_truststore_type and self._jdbc_truststore_type.upper() not in VALID_TRUSTSTORE_TYPES:
       ...
```

Reply:
Yes I think that will make it more clear. 👍